### ISSUE TITLE
Crashes Minecraft on Nintendo Switch consoles upon starting the client.

### ISSUE THREAD

**Issue by DelxHQ** [GeyserMC/GeyserOptionalPack#17]
It seems this pack (sometimes) crashes Minecraft on Nintendo Switch consoles upon starting the game.

I also think it sometimes crashes upon leaving the server.

**Comment by Camotoy** [comment]
"Starting the game" can you elaborate on what this means?

**Comment by DelxHQ** [comment]
Starting the client from the Switch home menu

**Comment by Camotoy** [comment]
That doesn't make much sense since you cannot install packs onto Switch? Sounds like you just need to reinstall Minecraft.

**Comment by DelxHQ** [comment]
I have a DNS server running that allows me to connect to Geyser if I join TheHive. The client seems to automatically download the pack when joining Geryser.

**Comment by Camotoy** [comment]
Bedrock is known for having the term "Bugrock", but even so that only adds the pack to your cached packs, not your installable/loaded packs.

**Comment by DelxHQ** [comment]
Makes sense, although it has happened to multiple players of mine. I guess Switch is the most buggy version of bedrock 🤷‍♀️
